### DISCUSSION TITLE
Необфусцированные js-файлы

### DISCUSSION THREAD

**Discussion by kirillkhaitbaev** [eigenein/bestmobabot#30] [Q&A] [answered]
У вас остались? Хотелось бы посмотреть, чтобы понимать как все устроено, когда смотришь нынешний код.

**Comment by eigenein** [comment, marked as answer]
Привет!

Покопайся в истории коммитов вот этой папки: https://github.com/eigenein/bestmobabot/commits/master/bestmobabot/js, куда я коммитил исходники игры.

Вот это похоже на самую первую версию, которую я использовал: https://raw.githubusercontent.com/eigenein/bestmobabot/5913dceda14c666a187ce34ea5f3e84b48f3c72c/bestmobabot/js/heroes.js
